RxSwift/Traits/Completable.swift

```diff
@@ -76,27 +76,46 @@ extension PrimitiveSequenceType where Trait == CompletableTrait, Element == Swif
      
      - parameter onCompleted: Action to invoke upon graceful termination of the observable sequence.
      - parameter onError: Action to invoke upon errored termination of the observable sequence.
+     - parameter onDisposed: Action to invoke upon any type of termination of sequence (if the sequence has
+     gracefully completed, errored, or if the generation is canceled by disposing subscription).
      - returns: Subscription object used to unsubscribe from the observable sequence.
      */
-    public func subscribe(onCompleted: (() -> Void)? = nil, onError: ((Swift.Error) -> Void)? = nil) -> Disposable {
+    public func subscribe(onCompleted: (() -> Void)? = nil,
+                          onError: ((Swift.Error) -> Void)? = nil,
+                          onDisposed: (() -> Void)? = nil) -> Disposable {
         #if DEBUG
                 let callStack = Hooks.recordCallStackOnError ? Thread.callStackSymbols : []
         #else
                 let callStack = [String]()
         #endif
 
-        return self.primitiveSequence.subscribe { event in
+        let disposable: Disposable
+        if let disposed = onDisposed {
+            disposable = Disposables.create(with: disposed)
+        }
+        else {
```

Comment:
Change to `} else {` please (e.g. move up a line)
```suggestion
        else {
```